nixos/doc/manual/release-notes/rl-2511.section.md

```diff
@@ -125,6 +125,9 @@
 
 - The `wstunnel` module was converted to RFC42-style settings, you will need to update your NixOS config if you make use of this module.
 
+- The `webfontkitgenerator` package has been renamed to `webfont-bundler`.
+  `webfontkitgenerator` is an alias to `webfont-bundler`.
```

Comment:
This should go into `doc/release-notes/rl-2511.section.md`, without the `nixos/` prefix, as it affects nixpkgs users and not only NixOS

Reply:
Right, thanks, that makes more sense :smile: